### DISCUSSION TITLE
能支持验证码吗？

### DISCUSSION THREAD

**Discussion by ljun223** [xiaohucode/yidaRule#134] [Ideas]
能不能添加个验证码函数，就类似阅读里的getVerificationCode()函数，getCookie()函数。

这样的话有些源就能不必经常换cookie了。

![Screenshot_2024-04-03-09-23-01-991_io.legado.app.release-edit.jpg](https://github.com/xiaohucode/yidaRule/assets/73452525/85664131-0734-40b4-9c9c-133ac4f7a048)

![Screenshot_2024-04-03-09-21-15-964_io.legado.app.release-edit.jpg](https://github.com/xiaohucode/yidaRule/assets/73452525/b69c7a79-7d11-4e88-a13d-b70e8731df75)

像这样的

**Comment by xiaohucode** [comment]
下版本加

**Reply by ljun223** [reply]
大佬啊！新添加的getVerificationCode()函数是不是有问题啊？试了几遍都不成功，抓了一下包才发现是发送请求的时候没有请求头。

可改了代码，发现它既不能调用源请求头，也不支持单独添加请求头……

**Reply by xiaohucode** [reply]
支持单独加请求头，imgurl,JSON.stringify(headers)

**Reply by ljun223** [reply]
原来是这么添加的，和阅读一样了，感谢大佬！

**Comment by ywjang5** [comment]
大佬能帮忙做一个书源吗，也是有验证码的。我现在用的是iOS，现在好像就这个软件支持验证码。网址是https://m.or77.net

**Reply by ljun223** [reply]
看小说建议用阅读，源仓库有人写过这个网站了。
https://www.e-yck.top/yuedu/shuyuan/json/id/4982.json

**Reply by ywjang5** [reply]
大佬我现在用的是苹果，阅读没有苹果版本，所以用不了这个书源。苹果有个源阅读，但是源阅读不支持验证码

**Reply by ljun223** [reply]
我暂时没时间写源，你可以去QQ“香色闺阁”频道去问问，那里大佬多，也有很多益达源。
